src/app/visual/date-pipe/date-pipe-visual.component.html

```diff
@@ -1,52 +1,119 @@
 <div
   class="app-screenshot"
+  style="width:300px"
 >
+
+  <h2>
+    Date pipe
```

Comment:
Note: The changes to the date pipe visual were intended to be an upgrade to make the intent and manual testing of the pipe easier. I made the fuzzy date pipe this way, and decided to give it a whirl on the regular date pipe. Let me know what you think!

**Note: There are no visual tests for both the date pipe and fuzzy date pipe. All coverage is done in unit.**